### ISSUE TITLE
Prints info and warning messages that aren't file-specific, for every file

### ISSUE THREAD

**Issue by dobesv** [robatwilliams/es-compat#24]
It prints this complaint for every file on the command line:

```
es-compat: No compatibility data for target family 'op_mini'
es-compat: checking compatibility for targets {
  chrome_android: '86',
  uc_android: '12.12',
  chrome: '84',
  edge: '85',
  firefox: '81',
  safari_ios: '10.0-10.2',
  opera: '71',
  safari: '13.1',
  samsunginternet_android: '12.0'
}
```

Ideally it would print it once, or not at all.

**Comment by robatwilliams** [comment]
The 2nd one is not a problem but it still logs for every file, needs fixing.

**Comment by robatwilliams** [comment]
@dobesv thanks for reporting, fixed in v2.0.0 - just released
